api/transfer.py

```diff
@@ -98,6 +99,29 @@ async def post(self, request) -> JSONResponse:
         return JSONResponse(credentials_and_id)
 
 
+class AssociationsImport(HTTPEndpoint):
+    async def post(self, request) -> JSONResponse:
+        project_id = request.path_params["project_id"]
+        request_body = await request.json()
+        user_id = request_body["user_id"]
+        auth_manager.check_project_access_from_user_id(user_id, project_id)
+        associations = request_body["associations"]
+        indices = request_body["indices"]
+        name = request_body["name"]
+        label_task_name = request_body["label_task_name"]
+        source_type = request_body["source_type"]
```

Comment:
personal preference: why are these stored in variables when the dict key is = to the variable name & only used once for a function call?

Reply:
true, changed that